README.md

```diff
@@ -42,30 +42,80 @@ easiest way to integrate VeraSDK with an existing `xcodeproj`.
 import VeraSDK
 ```
 
-2. Create a configuration object.
+2. Create a configuration object. Most fields are optional, check the [example integration](https://github.com/resonai/vera-ios-sdk/blob/main/Examples/VeraSDKExample-CP/VeraSDKExample-CP/TestSizeViewController.swift) for more parameters.
 
 ```swift
-let config = VeraConfiguration(
-    app: .init(clientID: "vera_client_app"),
-    user: .init(username: nil),
-    eventHandler: { event in
-        switch event {
-        case .refreshToken:
-            print("refresh the token")
-        @unknown default:
-            fatalError()
-        }
-    }
+Vera.useConfig(
+    .init(
+        app: .init(
+            clientID: "vera_client_id"
+        )
+    )
 )
 ```
 
-3. Build an instance of `VeraViewController` using the configuration and present it.
+3. Build an instance of `VeraViewController` and present it.
 
 ```swift
-let vera = VeraViewController.build(config: config)
+let vera = Vera.getController()
 present(vera, animated: true)
 ```
 
+## Bi-directional Communication
+
+The SDK implements bi-directional communication between the Vera platform and the client application. Communication is done using events.
+
+### Sending Events
+
+In order to send an event to the SDK, implement the `Vera.handleEvent(_:)` method.
```

Comment:
I think you mean "call" rather than "implement"

Reply:
Yup, nice catch 👍